### ISSUE TITLE
Packets tracing

### ISSUE THREAD

**Issue by liorbiu** [fabric/fabric#1619]
Hi,

I'm using fabric to automate execution on multiple servers.
Is there a way to get data similar to nload using fabric?
I'm interested specially in the amount of MBs sent and received during the execution.

I'm using the latest version of fabric and run it under CentOS 7.3

**Comment by ploxiln** [comment]
I'm pretty sure no such feature exists in fabric, nor in paramiko (the ssh library).

**Comment by bitprophet** [comment]
No, Fabric doesn't track this; I don't believe Paramiko does in a useful fashion, _but_ it does have to track _some_ statistics for data sent/received (in order to do things like calculate when to rekey) so it's remotely possible that could be extended sometime.

I'd put this down as a "nice to have" but it wouldn't be super high priority unfortunately. Thanks for the request :)
